config/default.yml

```
@@ -413,6 +413,11 @@ Rails/FindEach:
     - select
     - lock
 
+Rails/FreezeTime:
+  Description: 'Prefer freeze_time over travel_to with argument current time.'
```

I think this reads a little better:
```suggestion
  Description: 'Prefer `freeze_time` over `travel_to` with an argument of the current time.'
```